### ISSUE TITLE
[css-display-3][css2][css-writing-modes-3] initial direction & writing-mode from root or body?

### ISSUE THREAD

**Issue by prjnt** [w3c/csswg-drafts#3260]
https://drafts.csswg.org/css-display-3/#containing-block says:
<blockquote>(For the initial containing block, the values are taken from the root element.)</blockquote>

Some CSS specs (such as [CSS2 §10.1](https://drafts.csswg.org/css2/visudet.html#x1)) say that the 'direction' of the initial containing block always comes from the root element; whereas others (such as [css-writing-modes-3](https://drafts.csswg.org/css-writing-modes-3/#icb), if I'm correctly understanding “propagated to”) say that it is instead taken from the body element in some circumstances, and that this does not affect the values of the root element (last sentence before §8.1).

This conflict should be resolved.

(Also, the first paragraph of §8 in css-writing-modes should be changed not to contradict the second paragraph, e.g. by adding "(other than in the case described in the following paragraph)".  That avoids people stopping reading after the first paragraph because they think they already have the answer to their question.)

**Comment by xfq** [comment]
See also: #3066

**Comment by fantasai** [comment]
Fair point. Added 'unless otherwise specified'. Lmk if that seems acceptable, @prjnt ?
